wolfcrypt/src/rsa.c

```diff
@@ -3174,6 +3187,7 @@ static int wc_RsaFunction_ex(const byte* in, word32 inLen, byte* out,
     else
 #endif
     {
+        WOLFSSL_MSG("Calling wc_RsaFunctionSync...");
```

Comment:
Why have this?

Reply:
There was a large section of code where the embedded app would get stuck.

This was a breadcrumb, seemed useful at the time, but I've removed it.